libc-test/build.rs

```diff
@@ -4578,6 +4578,10 @@ fn test_haiku(target: &str) {
     });
 
     cfg.skip_fn(move |name| {
+        // private system calls
```

Comment:
I remember a similar discussion happened somewhere but private items shouldn't be exposed here, I think.

Reply:
do you mean in the crate or in build.rs ?

Reply:
From the crate, I mean, as it's basically unstable.

Reply:
oh ok